build.sbt

```diff
@@ -420,13 +421,15 @@ lazy val compiler = configureAsSubproject(project)
     scalacOptions in Compile in doc ++= Seq(
       "-doc-root-content", (sourceDirectory in Compile).value + "/rootdoc.txt"
     ),
-    Osgi.headers +=
+    Osgi.headers ++= Seq(
       "Import-Package" -> ("jline.*;resolution:=optional," +
                            "org.apache.tools.ant.*;resolution:=optional," +
                            "scala.util.parsing.*;version=\"${range;[====,====];"+versionNumber("scala-parser-combinators")+"}\";resolution:=optional," +
                            "scala.xml.*;version=\"${range;[====,====];"+versionNumber("scala-xml")+"}\";resolution:=optional," +
                            "scala.*;version=\"${range;[==,=+);${ver}}\"," +
                            "*"),
+      "Class-Path" -> "scala-reflect.jar scala-library.jar"
```

Comment:
In short : because this is what to expect from a JSR-223 compliant language. It is still useful when run from the distribution as obtained from the website. In cases where the directory layout is not right, it costs nothing to have the attribute, it will just not be used.